.gitmodules

```diff
@@ -124,3 +124,9 @@
 [submodule "libraries/cmake/source/augeas/gnulib/src"]
 	path = libraries/cmake/source/augeas/gnulib/src
 	url = https://github.com/osquery/third-party-gnulib
+[submodule "libraries/cmake/source/dbus/src"]
+	path = libraries/cmake/source/dbus/src
+	url = https://gitlab.freedesktop.org/dbus/dbus.git
+[submodule "libraries/cmake/source/expat/src"]
```

Comment:
osquery includes [libxml2](https://github.com/GNOME/libxml2), do we need both xml parsing libraries?
(It looks like libxml2 is only used in a single place)

Reply:
I can take a look into transferring dbus to use libxml2.

Reply:
Talking in office hours, it sounds like it's _hard_ to use libxml2 because dbus uses the `expat`. While I'm kinda sad, that seems fine? Might be the lesser of evils.